Can't run Pythagorean Triplet tests (compilation error)

PythagoreanTripletTest.java fails to compile due to duplicate method declarations:

> [ERROR] COMPILATION ERROR : 
[ERROR] /tmp/solution/src/test/java/PythagoreanTripletTest.java:[100,17] method tripletsWhoseSumIs840() is already defined in class PythagoreanTripletTest
[ERROR] /tmp/solution/src/test/java/PythagoreanTripletTest.java:[137,17] method tripletsWhoseSumIs30000() is already defined in class PythagoreanTripletTest

--- a/exercises/practice/pythagorean-triplet/src/test/java/PythagoreanTripletTest.java
+++ b/exercises/practice/pythagorean-triplet/src/test/java/PythagoreanTripletTest.java
@@ -97,7 +97,7 @@ public class PythagoreanTripletTest {
 
     @Ignore("Remove to run test")
     @Test
-    public void tripletsWhoseSumIs840() {
+    public void tripletsWhoseSumIs840WithFactorsLessThanOrEqualTo370() {
         List<PythagoreanTriplet> actual
                 = PythagoreanTriplet
                         .makeTripletsList()
@@ -134,7 +134,7 @@ public class PythagoreanTripletTest {
 
     @Ignore("Remove to run test")
     @Test
-    public void tripletsWhoseSumIs30000() {
+    public void tripletsWhoseSumIs30000WithFactorsLessThanOrEqualTo12500() {
         List<PythagoreanTriplet> actual
                 = PythagoreanTriplet
                         .makeTripletsList()

--- a/exercises/practice/sgf-parsing/src/test/java/SgfParsingTest.java
+++ b/exercises/practice/sgf-parsing/src/test/java/SgfParsingTest.java
@@ -128,7 +128,7 @@ public class SgfParsingTest {
     @Ignore("Remove to run test")
     public void escapedProperty() throws SgfParsingException {
         String input = "(;A[\\]b\nc\nd\t\te \n\\]])";
-        SgfNode expected = new SgfNode(Map.of("A", List.of("]b\nc\nd  e \n]")));
+        SgfNode expected = new SgfNode(Map.of("A", List.of("]b\nc\nd\t\te \n]")));
         SgfNode actual = new SgfParsing().parse(input);
         assertEquals(expected, actual);
     }

